MantidPlot/src/Mantid/FirstTimeSetup.cpp

```diff
@@ -156,6 +156,7 @@ void FirstTimeSetup::openManageUserDirectories()
   ad->setAttribute(Qt::WA_DeleteOnClose);
   ad->show();
   ad->setFocus();
+  // cppcheck-suppress memleak
```

Comment:
qt handles it.  it is owned by it's parent (the form opening it) and is deleted on closing that form.

It is a bit secret, but it is the qt standard way of doing things